Emerald Insight.js

```diff
@@ -0,0 +1,240 @@
+				"issue": "3/4",
+				"libraryCatalog": "Emerald Insight",
+				"pages": "192-199",
+				"publicationTitle": "International Journal of Prisoner Health",
+				"url": "https://doi.org/10.1108/IJPH-07-2016-0028",
+				"volume": "13",
+				"attachments": [
+					{
+						"title": "Full Text PDF",
+						"mimeType": "application/pdf"
+					}
+				],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.emerald.com/insight/search?q=testing&advanced=true&openAccess=true",
+		"items": "multiple"
+	},
+	{
+		"type": "web",
+		"url": "https://www.emerald.com/insight/content/doi/10.1108/S1085-462220150000016007/full/html",
+		"items": [
+			{
+				"itemType": "bookSection",
+				"title": "Creating a Cheat-Proof Testing and Learning Environment: A Unique Testing Opportunity for Each Student",
+				"creators": [
+					{
+						"lastName": "Menk",
+						"firstName": "K. Bryan",
+						"creatorType": "author"
+					},
+					{
+						"lastName": "Malone",
+						"firstName": "Stephanie",
+						"creatorType": "author"
+					}
+				],
+				"date": "January 1, 2015",
+				"ISBN": "9781784415877 9781784415884",
+				"abstractNote": "AbstractOriginality/value\nThis technique creates opportunities for students to have unique assignments encouraging student to student teaching and can be applied to assignments in any accounting course (undergraduate and graduate). This testing method has been used in Intermediate I and II, Individual Taxation, and Corporate Taxation.",
```

Comment:
Maybe it is cleaner to take here
```js
ZU.cleanInternal(doc.getElementById('abstract').textContent);
```
for the abstract. See also https://www.emerald.com/insight/content/doi/10.1108/07419051111154758/full/html for more subcontent under abstract.

Reply:
Done (won't use this when scraping from PDF, which I deemed acceptable)